src/OSPSuite.Core/Serialization/SimModel/Services/SimulationExportCreator.cs

```diff
@@ -14,7 +14,7 @@
 
 namespace OSPSuite.Core.Serialization.SimModel.Services
 {
-   public interface ICreateExportModelVisitor
+   public interface ISimulationExportCreator
```

Comment:
I have renamed this guy because the fact this is is a visitor is an implementation detail